overlord/hookstate/ctlcmd/is_connected.go

```diff
@@ -0,0 +1,101 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package ctlcmd
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/overlord/ifacestate"
+)
+
+type isConnectedCommand struct {
+	baseCommand
+
+	Positional struct {
+		PlugOrSlotSpec string `positional-args:"true" positional-arg-name:":<plug|slot>"`
+	} `positional-args:"yes"`
+}
+
+var shortIsConnectedHelp = i18n.G(`The is-connected returns zero if the given plug or slot is connected, and non-zero otherwise`)
+var longIsConnectedHelp = i18n.G(`
+The is-connected returns zero if the given plug or slot of the calling snap is
+connected, and non-zero otherwise.
```

Comment:
"the is-connected *command* returns..."
Also I think it's clearer to talk about success / failure rather than zero and non-zero, as above.